pages/common/timew.md

```diff
@@ -1,7 +1,7 @@
 # timew
 
 > A time tracking tool used to measure the duration of activities.
-> More information: <https://taskwarrior.org/docs/timewarrior>.
+> More information: <https://manned.org/timew>.
```

Comment:
Maybe we could use this?
```suggestion
> More information: <https://timewarrior.net/docs>.
```

Reply:
Yeah, sure. No idea how I didn't find that